composer.json

```diff
@@ -98,6 +104,9 @@
         ]
     },
     "config": {
+        "platform": {
+            "php": "7.1.3"
```

Comment:
This ensures that all dependencies are compatible with php 7.1.3, and not just the current system platform when generating the lock file.